iscdhcp/files/systemd_file.RedHat

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=DHCPv4 Server Daemon
+Documentation=man:dhcpd(8) man:dhcpd.conf(5)
+Wants=network-online.target
+After=network-online.target
+After=time-sync.target
+
+[Service]
+EnvironmentFile=-/etc/sysconfig/dhcpd
+Type=notify
+ExecStart=/usr/sbin/dhcpd -f -cf /etc/dhcp/dhcpd.conf -user dhcpd -group dhcpd --no-pid $DHCPDARGS
```

Comment:
shouldn't it be ``{{ ' '.join(salt['pillar.get']('iscdhcp:listen_interfaces', [])) }}`` instead of ``$DHCPDARGS``?